tools/xcelium_verilog_fixup.py

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright (c) 2014-2021, Lars Asplund lars.anders.asplund@gmail.com
+
+"""
+Perform necessary modifications to VUnit Verilog code to support
+Cadence Xcelium
+"""
+
+import os
+import re
+from pathlib import Path
+
+
+def replace_stop_by_finish(file_name):
+    """
+    Replace $stop by $finish
+    """
+
+    with Path(file_name).open("r", encoding="iso-8859-1") as fptr:
+        text = fptr.read()
+
+    text = text.replace("$stop(", "$finish(")
+
+    with Path(file_name).open("w", encoding="iso-8859-1") as fptr:
+        fptr.write(text)
+
+
+def add_finish_after_error(file_name):
+    """
+    Add $finish after a $error
+    """
+
+    with Path(file_name).open("r", encoding="iso-8859-1") as fptr:
+        text = fptr.read()
+
+    text = re.sub(r"(\$error\(.*\))", "\\1; $finish(1)", text)
+
+    with Path(file_name).open("w", encoding="iso-8859-1") as fptr:
+        fptr.write(text)
+
+
+def main():
+    """
+    Remove xcelium incompatabilities from source code
+    """
```

Comment:
```suggestion
    """Remove Xcelium incompatibilities from source code."""
```

Reply:
The style used throughout the rest of VUnit is what we used.

Reply:
Removing file.